modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/NodeJSServerCodegen.java

```diff
@@ -76,27 +82,13 @@ public NodeJSServerCodegen() {
         additionalProperties.put("apiVersion", apiVersion);
         additionalProperties.put("serverPort", serverPort);
 
-        /*
-         * Supporting Files.  You can write single files for the generator with the
-         * entire object tree available.  If the input file has a suffix of `.mustache
-         * it will be processed by the template engine.  Otherwise, it will be copied
-         */
-        // supportingFiles.add(new SupportingFile("controller.mustache",
-        //   "controllers",
-        //   "controller.js")
-        // );
-        supportingFiles.add(new SupportingFile("swagger.mustache",
-                        "api",
-                        "swagger.yaml")
-        );
-        writeOptional(outputFolder, new SupportingFile("index.mustache", "", "index.js"));
-        writeOptional(outputFolder, new SupportingFile("package.mustache", "", "package.json"));
-        writeOptional(outputFolder, new SupportingFile("README.mustache", "", "README.md"));
-        if (System.getProperty("noservice") == null) {
-            apiTemplateFiles.put(
-                    "service.mustache",   // the template to use
-                    "Service.js");       // the extension for each file to write
-        }
+        cliOptions.add(CliOption.newBoolean(GOOGLE_CLOUD_FUNCTIONS,
+            "When specified, it will generate the code which runs within Google Cloud Functions "
+                + "instead of standalone Node.JS server."));
```

Comment:
Great idea. Done.  Also added a section in README.mustache template, so people can refer to the URL after the code is generated.